build.py

```diff
@@ -359,6 +359,37 @@ def get_path(self):
 
 Project.add(Tool_nuget())
 
+class Tool_perl(Tool):
+    def __init__(self):
+        Tool.__init__(self,
+            'perl',
+            archive_url = 'https://dl.hexchat.net/misc/perl/perl-5.20.0-x64.tar.xz',

```

Comment:
I definitely do not like having this from the hexchat server.... can we include it in our github?